File: tests/fixtures/kotlin-library/src/main/kotlin/library/models/Genre.kt
package library.models

/** Genre categories for library books. */
enum class Genre {
    FICTION,
    NON_FICTION,
    SCIENCE,
    HISTORY,
    BIOGRAPHY;

    /** Human-readable label for display. */
    fun label(): String = name.replace("_", " ").lowercase()
        .replaceFirstChar { it.uppercase() }
}

File: tests/fixtures/python-library/library/models/genre.py
from enum import Enum


class Genre(Enum):
    """Genre categories for library books."""

    FICTION = "fiction"
    NON_FICTION = "non_fiction"
    SCIENCE = "science"
    HISTORY = "history"
    BIOGRAPHY = "biography"

    def label(self) -> str:
        """Human-readable label for display."""
        return self.value.replace("_", " ").title()
